server/etcdserver/api/membership/cluster_test.go

```diff
@@ -975,3 +978,48 @@ func TestIsReadyToPromoteMember(t *testing.T) {
 		}
 	}
 }
+
+/*
+covered by TestV2DeprecationSnapshotMatches
+// TestMembershipStore tests code path used by snapshot
```

Comment:
What's the plan with this test? Having both e2e and unit test is ok.

Reply:
I had added this test to simulate 3.5 style recovery. But now it is getting in the way of removing SetStore (and v2store from RaftCluster). I am investigating modification to the test or will remove it.